MAINTENANCE.md

```diff
@@ -0,0 +1,75 @@
+# Issue Tsar Role
+This document formally defines the Issue Tsar role. This is a repository maintenance role that is assigned to core contributors on rotating basis.
+
+## Responsibilities
+Issue Tsar is responsible for:
+- Health of the CI, nightly releases and benchmark infrastructure.
+- PRs of external contributors: assigning someone to review, or handling themselves.
+- Triaging issues (especially new):
+  - Each issue needs to be assigned an `itype` and 1 or more `area` labels.
+  - Where applicable, new issues need to be designated for the Spree or Semester projects with the corresponding labels.
+  - Regressions from an earlier Scala 3 release must be classified with the “regression” label and assigned to the next release’s milestone.
+  - Modifying issue labels to best capture information about the issues
+    - Attempting to reproduce the issue (or label “stat:cannot reproduce”)
+    - Further minimising the issue or asking the reporter of the issue to minimize it correctly (or label “stat:needs minimization”)
+
+Other core teammates are responsible for providing information to the Issue Tsar in a timely manner when it is requested if they have that information.
+
+## Assignment
+Issue Tsar is appointed for 7 days and is responsible for what is specified in the “Responsibilities” section during those 7 days. Their assumption of the role starts from the Dotty Meeting on Monday and ends on the next Dotty Meeting on Monday.
```

Comment:
I'm still wondering if it is not better to en on Friday. This way the new person has the time to fully process issues that came on the weekend. But this could be changed later.

Reply:
I'm also in favor of this, but during the meeting, the majority seemed to be in favor of the old approach. We can see how it goes and change it in the future if need be.